tests/functional-swap.rs

```diff
@@ -166,9 +310,10 @@ async fn run_swap(
         .unwrap();
 
     // run until bob has the btc funding address
-    let address = retry_until_bitcoin_funding_address(cli_bob_progress_args.clone());
+    let address = retry_until_bitcoin_funding_address(cli_bob_progress_args.clone()).await;
 
     // fund the bitcoin address
+    let lock = execution_mutex.lock().await;
     let amount = bitcoin::Amount::ONE_SAT * 100000150;
```

Comment:
Can use `Amount::from_sat()`

Reply:
zkao said the same thing yesterday ^^ . I will fix all the amounts once your progress PR is in, then I can finally consume the progress messages for them and don't need to do any of this constant amount handling.